SpatialGDK/Source/Private/Interop/GlobalStateManager.cpp

```diff
@@ -29,6 +34,45 @@ void UGlobalStateManager::ApplyData(const Worker_ComponentData& Data)
 	SingletonNameToEntityId = GetStringToEntityMapFromSchema(ComponentObject, 1);
 }
 
+void UGlobalStateManager::ApplyMapData(const Worker_ComponentData& Data)
+{
+	Schema_Object* ComponentObject = Schema_GetComponentDataFields(Data.schema_type);
+	
+	// Set the Map URL.
+	if(Schema_GetObjectCount(ComponentObject, 1) == 1)
+	{
+		SetDeploymentMapURL(GetStringFromSchema(ComponentObject, 1));
+	}
+
+	if(Schema_GetBoolCount(ComponentObject, SpatialConstants::GLOBAL_STATE_MANAGER_ACCEPTING_PLAYERS_ID) == 1)
+	{
+		// Set the AcceptingPlayers state.
+		bool bDataAcceptingPlayers = bool(Schema_GetBool(ComponentObject, SpatialConstants::GLOBAL_STATE_MANAGER_ACCEPTING_PLAYERS_ID));
```

Comment:
static_cast<>

Reply:
You need to do `!!` I believe, static cast wont compile as it will generate a warning.

Reply:
What's the advantage of these over the c-style cast? I think this is the most readable.